aws_doc_sdk_examples_tools/config/services.yaml

```diff
@@ -1934,6 +1934,22 @@ lightsail:
   tags:
     product_categories: {}
   version: lightsail-2016-11-28
+location:
+  apiref: location/latest/APIReference/Welcome.html
+  blurb: Use Amazon Location Service to easily and securely add maps, places, routes, geofences, and trackers, to your applications. With Amazon Location Service, you safeguard customer data, reduce cost, and easily integrate with other AWS services.
+  expanded:
+    long: Amazon Location Service
+    short: Amazon Location
+  guide:
+    subtitle: Deverloper Guide
+    url: location
+  long: '&ALlong;'
+  short: '&ALshort;'
+  sort: Location
+  tags:
+    product_categories: {}
+  version:
+    location-2020-11-19
```

Comment:
This is only one of at possibly four subclients. The API reference linked above shows just three v2 clients (all prefixed with geo-), none of which implement the entirety of what's in this client. It looks like this service was perhaps split into several subclients (or is in process of doing so?). Presumably, examples should use v2 clients when they exist, which would mean creating a bundle for the services.

Reply:
From my research, LocationAsyncClient offers a broad range of functionalities for AWS Location Service, while GeoRoutesAsyncClient (etc) focuses specifically on more advanced features. Neither client is deprecated; they are designed to address different aspects of AWS Location Service based on your application's requirements. I am double checking with SDK team.

Reply:
Added the full suite under the "Geo" bundle